### ISSUE TITLE
clarification on return mailbox audit log events in audit log searches

### ISSUE THREAD

**Issue by limjianan** [MicrosoftDocs/microsoft-365-docs#1094]
Only users with E5 licenses or mailboxes where mailbox audit logging was manually enabled by an admin will return mailbox audit log events in audit log searches in the Security & Compliance Center or via the Office 365 Management Activity API. &lt;--- this statement,  means, only user that is E5 licence will return mailbox audit log events.  OR "mailboxes where mailbox audit logging was manually enabled by an admin " will return mailbox audit log events. 

Since the Top of the statement said it will turn on mailbox audit logging by default; do this means it will not return mailbox audit log event if it is not E5? Further, it did not clarify it is Office 365 E5 

I believe since it is logged since 2019 January, it should be returned unless it is disabled manually.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b1aac99b-5c65-74d3-7610-b7ba4fcac493
* Version Independent ID: a0b2ab87-7d52-3429-630e-6b7a316b5f23
* Content: [Manage mailbox auditing - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/enable-mailbox-auditing#more-information)
* Content Source: [microsoft-365/compliance/enable-mailbox-auditing.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/enable-mailbox-auditing.md)
* Service: **o365-seccomp**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@limjianan, I've updated the topic to clarify on some of your points.

Auditing on by default is E5 only; everyone else who isn't E5 needs to turn it on manually on each mailbox. After either of those two conditions are satisfied (E5 or manually enabled), only then will audit log events appear in Security & Compliance Center audit log searches or in the Office 365 Management Activity API.

To put it another way, does the user have an E5 license? If not, don't expect any audit log events in SCC audit log searches or in the O365 Management Activity API. If the user has an E3 or some other license other than E5, have you manually enabled audit logging on their mailbox? If not, don't expect any audit log events in SCC audit log searches or in the O365 Management Activity API.

Microsoft 365 E5 and Office 365 E5 are identical when it comes to mailboxes and auditing.

Thanks for the feedback.

**Comment by anwarmahmood1** [comment]
This still isn't precisely accurate.
I have "Office 365 A1 Plus for Faculty".  With AuditEnabled explicitly set to true, events did not appear in the unified audit log.  When I also switched to "Microsoft 365 A3 for students use benefit", then events did appear in the unified audit log.

Hence "everyone else" needs to be more specific.  I've wasted a week on a Microsoft Premier support call only to resolve this myself.
